marketplace/google-analytics/extension.json

```diff
@@ -0,0 +1,32 @@
+{
+  "id": "google-analytics",
+  "name": "google-analytics",
+  "majorVersion": 1,
```

Comment:
Just noticed it. I wouldn't add new properties that are incompatible with API in the extension.json. Can you use `package.json` version for this purpose?